protos/sources/vega/data_source.proto

```diff
@@ -39,7 +39,7 @@ message DataSourceDefinitionExternal {
   // Types of External data sources
   oneof source_type {
     DataSourceSpecConfiguration oracle = 1;
-    EthCallSpec eth_call = 2;
+    EthCallSpec eth_oracle = 2;
```

Comment:
Docs please! :)

Reply:
this is merge to feature branch for us to coordinate work - there will be a PR to push this to develop in which we will be sure to sort the docs, its all a bit in flux at the moment as to how we exactly define these specs